static/index.js

```diff
@@ -3,6 +3,17 @@ import ReactDOM from "react-dom";
 import App from "../src/components/App";
 import "./index.css";
 
+const activeTheme = chrome.devtools.panels.themeName;
+
+if (activeTheme === 'light') {
+  // using require b/c import can not be synchronously dynamically loaded
+  require('./themes/light.css');
+}
+
+if (activeTheme === 'dark') {
+  require('./themes/dark.css');
+}
```

Comment:
I tried a few more approaches here (that were syntactically elegant) but they wouldn't work. webpack doesn't like dynamic require paths, and dynamic imports are async. I could wrap a dynamic import an an async() func but i thought this might be more straightforward (?)

lmk your thoughts!

Reply:
Will this allow for live reloading if a users changes their preference?

Reply:
i think this approach won't! good call. I looked into how React handles this, and i'll try to emulate it https://github.com/facebook/react/blob/3cc564547c81b6e9a3bd6a04aa0725e668ca6559/packages/react-devtools-extensions/src/main.js#L171

Reply:
alright @jbaxleyiii I confirmed that the entire panel is rebuilt when the theme setting is changed, and the new theme is applied